docs/bdd/index.rst

```diff
@@ -0,0 +1,24 @@
+The BDD Guide
+=============
+
+Behaviour driven development is an approach to software development process that provides software development and management teams
+with shared tools and a shared process to collaborate on software development. The awesome part of BDD is its ubiquitous language,
+which is used to describe the software in English-like sentences of domain specific language.
+
+The application's behaviour is described by scenarios, and those scenarios are turned into automated test suites with tools such as Behat.
+
+Sylius behaviours are fully covered with Behat scenarios. There are more than 1200 scenarios in the Sylius suite, and if you want
+to understand some aspects of Sylius better, or are wondering where to click to achieve something, we strongly recommend
```

Comment:
I am not sure about this `wondering where to click to achieve something` our scenarios describe behavior of the system.